Exception handling.js

```diff
@@ -1,19 +1,19 @@
-
-
-try {
-    let a = 3;
-    console.log(a);
-
-
-} catch(e) {
-    alert('let перед использованием нужно объявить')
-
-}
-
-try{
-    let b=1/0
-    console.log(b)
-
-}catch (e) {
-    alert('на ноль делить нельзя');
-}
+
+
+try {

```

Comment:
this function works well, you do not generate an Error, try to switch console.log string and let a = 3 string

Reply:
No, this is not correct, there is no error generation here.